src/nft/pages/collection/index.tsx

```diff
@@ -60,83 +62,85 @@ const Collection = () => {
   }
 
   return (
-    <>
-      <Column width="full">
-        {contractAddress ? (
-          <>
-            {' '}
-            <Box width="full" height="160">
+    <Trace page={PageName.NFT_COLLECTION_PAGE} properties={{ contractAddress }} shouldLogImpression>
```

Comment:
Right now this wraps the collection page but also the error `no collection page exists here` page. Wanted to see if this is intended or if this wrapper should go after the `contractAddress ?` condition

Reply:
I removed the Trace event here as its no longer needed (fired at App level for all routes)